utils/utils.ts

```diff
@@ -46,6 +46,11 @@ export function fromStringToUnitString(value: string) {
   return fromBNToUnitString(valueBN);
 }
 
+export function fromStringToBN(value: string) {
```

Comment:
there is an stringToBN function, that handles hex results as well, plz reuse that one :)